test/config.js

```diff
@@ -49,6 +49,15 @@ test.cb('config.fatal = true', t => {
   });
 });
 
+test.cb('config.fatal = true with an exec() failure includes a .code on the Error', t => {
```

Comment:
Change this to a regular `test(`, not a `test.cb(`. You only need `test.cb(` when you're calling an async function which has a callback. In your case, you don't want that: you are testing the synchronous version of `shell.exec()`.